src/main/resources/application-dev.yaml

```diff
@@ -0,0 +1,22 @@
+management:
```

Comment:
We should be able to delete this file now, as we're not using ActiveMQ anywhere (for which this was the original purpose in other repos)